backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java

```diff
@@ -36,4 +42,9 @@
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
+
+    @GetMapping("/token/{shopId}")
+    public List<Token> getTokens(@PathVariable UUID shopId) {
+		return virtualQueueService.getTokens(shopId);
+	}
```

Comment:
Similarly this closing bracket should be in line with the previous closing bracket (on line 44)

Reply:
It's showing inline in my local code. Did u check by cloning it on your local machine?

Reply:
Yeah my bad. Cloned it and everything seems alright in my local machine.